google/cloud/spanner_v1/pool.py

```diff
@@ -23,6 +23,7 @@
 
 
 _NOW = datetime.datetime.utcnow  # unit tests may replace
+_SESSION_EXPIRY_DURATION = datetime.timedelta(hours=1)
```

Comment:
Is this the time interval between two consecutive `ping()`? Then, I think `50mins` would be better.

Reply:
Ignore my previous comment. I see this is the expire duration for a session.

Reply:
This is the time after which a session is considered expired as the backend will have GC'd it.

In the [constructor for PingingPool](https://github.com/googleapis/python-spanner/blob/a9bf3e579b5b8f64f9549a22d28f51ff8781fd83/google/cloud/spanner_v1/pool.py#L345) there is a `ping_interval` kwarg which is the time between two consecutive pings. The default for that is 3000 seconds ie 50 minutes.